src/mscorlib/src/System/String.CoreCLR.cs

```diff
@@ -64,6 +64,18 @@ public partial class String
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
         internal extern bool IsAscii();
 
+        // native call to COMString::CompareOrdinalEx
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        internal static extern int CompareOrdinalHelper(String strA, int indexA, int countA, String strB, int indexB, int countB);
+
+        //This will not work in case-insensitive mode for any character greater than 0x7F.
+        //We'll throw an ArgumentException.
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        internal static extern unsafe int nativeCompareOrdinalIgnoreCaseWC(String strA, sbyte* strBBytes);
+
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        private static extern int InternalMarvin32HashString(string s);
```

Comment:
I am fixing this one in https://github.com/dotnet/coreclr/pull/17029.

Reply:
I noticed. I think moving whole GetHashCode (+deps) to String.cs is best here

Reply:
Why? It is same between CoreCLR and CoreRT.